client/src/components/pages/page.css

```diff
@@ -1,5 +1,12 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
```

Comment:
do you need this here when you have already defined it in app.css?

Reply:
Actually, I wasn´t sure about that. I found it hard to begin with, to figure out, where to define something like global css. But the plan is, to change everything to styled components in the next steps, anyway.
Thanks for review!